src/start/RealmScreen.js

```diff
@@ -53,6 +54,26 @@ class RealmScreen extends PureComponent<Props, State> {
 
     const { dispatch } = this.props;
 
+    let parsedRealm;
+    try {
+      parsedRealm = new WhatwgURL(realm);
+    } catch (err) {
+      this.setState({
+        progress: false,
+        error: 'Please enter a valid URL',
+      });
+      return;
+    }
+
+    if (parsedRealm.username !== '') {
+      this.setState({
+        progress: false,
+        error:
+          'Please enter the server URL, not your email. You can enter your credentials in the next screen.',
```

Comment:
I think most people don't know what "credentials" means in this context. Someone who enters an email address at the "Zulip server URL" prompt is especially unlikely to know what it means :wink:

So let's just drop that sentence.

(It's also not quite true -- often the next screen will be a menu of options for how to log in. Then the *next* screen after that might involve entering credentials, depending on the auth method chosen.)